guides/doc-Installing_Server_Disconnected/master.adoc

```diff
@@ -26,10 +26,6 @@ include::common/modules/proc_configuring-server-to-consume-content-from-a-custom
 
 include::common/assembly_configuring-inter-server-synchronization.adoc[leveloffset=+2]
 
-include::common/assembly_importing-kickstart-repositories.adoc[leveloffset=+2]
-
-include::common/assembly_importing-the-project-client-name-repository.adoc[leveloffset=+2]
-
```

Comment:
This wasn't here before, was it? Please drop this part of your PR.

Reply:
I think this is part of a different PR I'm doing. Not sure why this showed up.